docs/standard-library/projected-names.md

```diff
@@ -139,3 +150,30 @@ class CertificateAttributes
 </td>
 </tr>
 </table>
+
+## Order of Operations
```

Comment:
I am not sure we should have that doc in here. This is specific to json and client, so might more belong to the dpg area.

Or we need to make this clear it is just an example

Reply:
I agree it doesn't make much sense here. However, I'm not really sure why this is documentation at all instead of test scenarios in cadl-ranch or, as mentioned in the issue, an API in the DPG library that executes the right operations in the right order so each emitter doesn't have to duplicate it. @johanste and I were talking about that earlier.

Reply:
In the fullness of time, it should be part of DPG helpers, however, we need to document what the correct pattern is before we implement as a helper.

Also, I think we need to be explicit about how to use the projected names helpers correctly in each case, and when to execute them in relation to versioning projections